### DISCUSSION TITLE
Could you add and support the driver HX8347G?

### DISCUSSION THREAD

**Discussion by liuquanli1970** [Bodmer/TFT_eSPI#1708] [New processor/display support or capability request]
It's a great job. Thank you Bodmer!
I'm also designing some projects with esp32, but the LCD driver chip used is hx8347g(240*320resolution). I have a complete datasheet about it. I don't know if you can add and support this driver?
thank you!

**Comment by liuquanli1970** [comment]
[HX8347-G(T).pdf](https://github.com/Bodmer/TFT_eSPI/files/6433868/HX8347-G.T.pdf)

**Comment by Bodmer** [comment]
Support could be added but the display has a particular command sequence that would need to be debugged during the code adaptation and development. I am unable to do this as I do not have one of those displays.
